src/helpers/update-helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { ferdiumVersion } from '../environment-remote';
```

Comment:
These functions might be better in the `environment-remote` file? Just checking which is the better location. Also, have you added tests for these?

Reply:
Haven't added tests for this (actually don't know how to do that xD)

Reply:
@SpecialAro This should be quite simple to test actually (you can look at https://github.com/ferdium/ferdium-app/blob/develop/test/helpers/url-helpers.test.ts for example tests)

The only thing you might need to change a bit is getFerdiumVersion() and pass the `window.location.href` as a property, that way the function itself doesn't need any outside information and makes it unit testable. Hit me up on Discord if you need any help :)